src/App.vue

```diff
@@ -414,12 +412,11 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
     // set current date from "real time" date from server
     this.setCurrentDate(DateUtilities.dateToYyyyMmDd(this.getCurrentJsDate))
 
-    // get and store keycloak roles
+    // ensure user is authorized to access this business
     try {
-      const keycloakRoles = GetKeycloakRoles()
-      this.setKeycloakRoles(keycloakRoles)
+      await this.checkAuth()
     } catch (error) {
-      console.log('Keycloak error =', error) // eslint-disable-line no-console
```

Comment:
I deleted this and moved checkAuth() here instead -- I figure, do it as soon as possible so we know whether we should continue or not.